From 712cff175abfbf95428ed09a875a15f4047ce105 Mon Sep 17 00:00:00 2001
From: Kent Fredric <kentfredric@gmail.com>
Date: Sun, 11 Jun 2017 13:22:46 +1200
Subject: [PATCH] Fix "do" to forcibly use a relative path on Perl 5.26

Reliance on "." in @INC was previously how this worked. But that
becomes broken on Perl 5.26

Resolves RT#120711 ( Well, at least the Math-Pari parts, the pari
    parts themselves need additional fixing, and that's not viable
    to really perform in Math::Pari )

Bug: https://rt.cpan.org/Ticket/Display.html?id=120711
---
 test_eng/ex.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test_eng/ex.t b/test_eng/ex.t
index a1a2286..0ad57ee 100644
--- a/test_eng/ex.t
+++ b/test_eng/ex.t
@@ -10,5 +10,5 @@ $dir1 = "CHANGE_ME";
 $dir1 = "$dir/../$dir1" unless $dir1 =~ m|^([a-z]:)?[\\/]|i;
 @ARGV = "$dir1/src/test/$long_bits/$name";
 @ARGV = "$dir1/src/test/32/$name" unless -r $ARGV[0];
-do 'test_eng/Testout.pm';
+do './test_eng/Testout.pm';
 die if $@;
-- 
2.13.1

